pom.xml

```diff
@@ -193,15 +193,17 @@
             org.codehaus.groovy.*;resolution:=optional,\
             kotlin.*;resolution:=optional,\
             io.vertx.groovy.*;resolution:=optional,\
-            io.vertx.ext.auth,\
-            io.vertx.ext.auth.*;resolution:=optional,\
             io.vertx.lang.rxjava.*;resolution:=optional,\
+            io.vertx.lang.reactivex.*;resolution:=optional,\
             io.vertx.lang.groovy.*;resolution:=optional,\
             io.vertx.codegen.annotations;resolution:=optional,\
             io.vertx.rx.java;resolution:=optional,\
             io.vertx.rxjava.core.*;resolution:=optional,\
             io.vertx.rxjava.ext.auth.*;resolution:=optional,\
+            io.vertx.reactivex;resolution:=optional,\
```

Comment:
Cant' we merge both instructions into one:

io.vertx.reactivex.*;resolution:=optional.

Is there other required packages that must be mandatory ?

Reply:
Well, there are MQTT classes generated in `io.vertx.reactivex.mqtt`, that seems weird to optionally import something that is already inside of the bundle

Reply:
Is there a dependency on auth ? Some module relies on vertx-auth to handle the authentication.

Reply:
No code in this bundle use auth, It should never have been there I think.
My bad, in my initial contribution, I forgot to remove them ..